analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -99,20 +99,25 @@ abstract class PackageManager(
 
         /**
          * Recursively search the [directory] for files managed by any of the [packageManagers]. The search is performed
-         * depth-first so that root project files are found before any subproject files for a specific manager.
+         * depth-first so that root project files are found before any subproject files for a specific manager. Path
```

Comment:
Nit: Maybe add a concrete use-case to the commit message? ("To skip problematic / large projects that are irrelevant compliance-wise.")

Reply:
I extended the commit message slightly.